src-java/kilda-pce/src/test/java/org/openkilda/pce/AvailableNetworkFactoryTest.java

```diff
@@ -272,4 +433,14 @@ private static void assertAvailableNetworkIsCorrect(IslImmutableView isl, Availa
         assertNotNull(dst);
         assertEquals(1, dst.getIncomingLinks().size());
     }
+
+    private static void assertAvailableNetworkIsNotCorrect(IslImmutableView isl, AvailableNetwork availableNetwork) {
```

Comment:
Wrong name of the method. Actually network is correct. Here you are checking that these is no ISL in available network

Reply:
Done, checked it inside the test case